ci/get-ocp-repo.sh

```diff
@@ -1,205 +1,19 @@
 #!/bin/bash
 set -euo pipefail
 
```

Comment:
would be kind nice to still get a description here about what this script does.. i.e. why is it needed?

The previous version had a description.